### ISSUE TITLE
Please update doc with following helpful details re: DEM account limitations

### ISSUE THREAD

**Issue by davehop** [MicrosoftDocs/memdocs#2738]
Please update the article with detailed information to answer the following customer scenarios/questions...

(Q1) 
Is the 1000 enrolled device limit based on the **total** number of devices enrolled by one DEM account? Or is that the number of **currently** enrolled devices? For example:

- We enroll 500 devices using a DEM account
- Before enrolling any more devices, we retire all of those enrolled above.
- How many devices can now be enrolled by that same DEM account?  Only 500 more, or can that account now enroll 1000 again?

(Q2)
For the 150 DEM account per tenant limit:
- Is that 150 total lifetime of tenant or is that 150 concurrent/active DEM accounts?

Thank you.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7786d9c0-55b0-9bff-fc0a-f4ee70ee0615
* Version Independent ID: cf0c17a0-fdc7-e0af-db82-913e36d2ec7c
* Content: [Enroll devices using a device enrollment manager account - Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/enrollment/device-enrollment-manager-enroll)
* Content Source: [memdocs/intune/enrollment/device-enrollment-manager-enroll.md](https://github.com/MicrosoftDocs/memdocs/blob/main/memdocs/intune/enrollment/device-enrollment-manager-enroll.md)
* Service: **microsoft-intune**
* Sub-service: **enrollment**
* Technology: ****
* GitHub Login: @lenewsad
* Microsoft Alias: **lanewsad**

**Comment by davehop** [comment]
Hi, my feedback did not involve any typo's.  

I am asking for the article to be updated with answers to the above questions/scenarios regarding DEM account limitations

**Comment by msbemba** [comment]
Sorry about that. Another issue was tagged incorrectly. I'll get the answers and update the document shortly.

**Comment by msbemba** [comment]
@davehop Thank you for your feedback. I have made changes to the document via PR#https://github.com/MicrosoftDocs/memdocs/pull/2759 and waiting for the author to verify and approve.  Once it is approved, the updates will be live.

Thanks
Sri

**Comment by yogkumgit** [comment]
@msbemba Thank you very much for the contribution and updating the documentation with PR. @davehop Hope this update is helpful for you. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.
